src/System.Reflection.TypeExtensions/pkg/System.Reflection.TypeExtensions.pkgproj

```diff
@@ -6,19 +6,27 @@
       <SupportedFramework>net461;netcoreapp2.0;$(AllXamarinFrameworks)</SupportedFramework>
     </ProjectReference>
     <ProjectReference Include="..\src\System.Reflection.TypeExtensions.csproj" />
+    <HarvestIncludePaths Include="ref/net46;lib/net46" />
```

Comment:
nit: Some projects you list the InboxOnTargetFramework's before HarvestIncludePaths and others you do the reverse.